vassal-app/src/main/java/VASSAL/counters/GlobalCommand.java

```diff
@@ -817,6 +833,7 @@ public int hashCode() {
     result = prime * result
       + ((reportFormat == null) ? 0 : reportFormat.hashCode());
     result = prime * result + (reportSingle ? 1231 : 1237);
+    result = prime * result + (isSuppressSounds() ? 1231 : 1237); // No idea what is appropriate here, just copying the mysterious line above
```

Comment:
Oh wow, that is the way old school way of doing this. The entire body of this function could be replaced with this:
```
return Objects.hash(keyStroke, reportFormat, reportSingle, selectFromDeckExpression, suppressSounds);
```

Reply:
updated